src/activities/AddGeoreferenceToPdf.ts

```diff
@@ -0,0 +1,175 @@
+import type { IActivityHandler } from "@vertigis/workflow";
+import {
+    PDFArray,
+    PDFDict,
+    PDFDocument,
+    PDFName,
+    PDFNumber,
+    PDFString,
+} from "pdf-lib";
+
+export interface AddGeoreferenceToPdfInputs {
+    /**
+     * @description The source PDF document to add the georeference metadata to. Either a file path or byte content are accepted.
+     * @required
+     */
+    source: Blob;
+
+    /**
+     * @description The bounding coordinates of the map element to georeference in PDF user space.
```

Comment:
I see we already have a number of uses of _coordinate_ in other activities, but _co-ordinate_ would be better.

Reply:
I disagree. `coordinate` is acceptable English and is the spelling commonly used in GIS.

This also intentionally matches the productized activity.

Reply:
OK, let's keep it like that. After I wrote this comment I was surprised to notice the same. Almost everywhere I looked it was coordinate.